modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -228,7 +228,12 @@ private void doStartNode(final String defaultConfig) {
         Log.d(TAG, "Node config " + config);
 
         String res = Statusgo.StartNode(config);
-        Log.d(TAG, "StartNode result: " + res);
+        if (res.startsWith("{\"error\":\"\"")) {
+            Log.d(TAG, "StartNode result: " + res);
+        }
+        else {
```

Comment:
Do we really need this if/else?

Reply:
@rasom While looking at this bug, more than one person missed this critical entry in the log, so it felt like something was needed to avoid the same thing happening in the future. This differentiates this log entry in case it represents an error, and hopefully helps bringing it to the attention of the developer (e.g. if we apply different colors to different log entry levels).
I agree that the implementation is hacky and can definitely use an improvement. When you question the need for the `if`, is it tied to how it's implemented, or whether it is necessary at all?

Reply:
oh sorry, i missed d/e difference, was under impression that lines inside if/else branches are identical